spring-file/src/main/java/com/example/file/FileController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.file;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+public class FileController {
```

Comment:
also, I'd suggest to add 
- unit tests for the `FileService` and controller
- qulice to check codestyle

Reply:
Is it necessary to add qulice check style if the main purpose of this application is only benchmark?

Reply:
I'd say yes, it's even more important for benchmarks to be properly written and perfectly readable as in the case of mistake the results can be not valid... It's my opinion, however, ask Kirill if you have doubts.